internal/txapp/txapp.go

```diff
@@ -671,6 +666,8 @@ func (r *TxApp) AccountInfo(ctx context.Context, acctID []byte, getUncommitted b
 // largest nonce of the transactions included in the block that are authored by the proposer.
 // This transaction only includes voteBodies for events whose bodies have not been received by the network.
 // Therefore, there won't be more than 1 VoteBody transaction per event.
+//
+// @charitha: does this method need to return [][]byte or just []byte (one tx)?
```

Comment:
RIght now we only have 1 proposerTx, but might expand in future if we need to add any other kind of proposerTxs, so designed it to accommodate that.